database/postgresconn/postgresconn_test.go

```diff
@@ -156,40 +151,8 @@ func TestMultipleStatements(t *testing.T) {
 
 		// make sure second table exists
 		var exists bool
-		if err := d.(*Postgres).conn.QueryRowContext(context.Background(), "SELECT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = 'bar' AND table_schema = (SELECT current_schema()))").Scan(&exists); err != nil {
-			t.Fatal(err)
-		}
-		if !exists {
-			t.Fatalf("expected table bar to exist")
-		}
-	})
-}
-
-func TestMultipleStatementsInMultiStatementMode(t *testing.T) {
```

Comment:
No longer supporting "multi statement mode", reducing surface area of the library. We just rely on the streaming parser now.